modules/API/API.js

```diff
@@ -556,12 +556,17 @@ class API {
      * has been turned on/off.
      *
      * @param {boolean} on - True if screen sharing is enabled.
+     * @param {Object} details - Additional information about the screen
+     * sharing.
+     * @param {string} details.sourceType - Type of device or window the screen
```

Comment:
maybe we should also update the docs here:
https://github.com/jitsi/jitsi-meet/blob/master/doc/api.md
do you know what are the possible values there ?

Reply:
screen - electron
window - electron
device - spot wired screenshare
proxy - spot wireless screenshare
undefined - unknown, from normal desktop picking flow on browser